src/charts/c3/c3.mixin.js

```diff
@@ -0,0 +1,57 @@
+angular.module('patternfly.charts').factory('ChartsMixin', function() {
```

Comment:
> The version we have in our cfmePOC...

You'll have to excuse my ignorance but what is cfmePOC?

You can inject an [angular constant](https://docs.angularjs.org/api/auto/service/$provide#constant) as well.  To me it makes sense to make changing the title dynamic but all of these static config values make me think of a constant.  I'll have to take a deeper look at how this is used as I don't see where the title is set at first glance.